src/core/ext/filters/http/message_compress/message_compress_filter.cc

```diff
@@ -141,6 +159,8 @@ class CallData {
 
   grpc_core::CallCombiner* call_combiner_;
   grpc_compression_algorithm compression_algorithm_ = GRPC_COMPRESS_NONE;
+  int grpc_min_message_size_to_compress_;
```

Comment:
Please call this `min_message_size_to_compress_`.

Reply:
done